cores/rp2040/RP2040.h

```diff
@@ -118,89 +142,108 @@ class _MFIFO {
 
 class RP2040;
 extern RP2040 rp2040;
-class RP2040 {
+class RP2040
+{
 public:
-    RP2040() {
+    RP2040()
+    {
         _epoch = 0;
         // Enable SYSTICK exception
         exception_set_exclusive_handler(SYSTICK_EXCEPTION, _SystickHandler);
         systick_hw->csr = 0x7;
         systick_hw->rvr = 0x00FFFFFF;
     }
 
-    ~RP2040() { /* noop */ }
-
+    ~RP2040()
+    { /* noop */
+    }
 
     // Convert from microseconds to PIO clock cycles
-    static int usToPIOCycles(int us) {
+    static int usToPIOCycles(float us)
```

Comment:
Can you please make this two overloaded functions.  The original one with a `int us` and your addition with `float us` as a parameter instead.

Making it take a float always is very expensive in generated code and slow since in all cases but Servo it's taking an `int`.

Reply:
hmm alright. I'll probably have to change the parameter type of the overload to a double for that to work